_posts/2021-12-23-Fortran-lang-2021-in-review.md

```diff
@@ -0,0 +1,350 @@
+* [Thirumalai Shaktivel](https://github.com/Thirumalai-Shaktivel) (AST generation in LFortran)
+
+You can read in more detail about their projects
+[here](https://summerofcode.withgoogle.com/archive/2021/organizations/6542461173760000).
+
+A big thank you to all students, mentors, and administrators, for their great
+work, and of course, to the Google Summer of Code program for making it
+possible for students to participate.
+
+We plan to apply for Google Summer of Code in 2022 as well.
+The program is no longer limited to students and anybody 18 or older can apply
+to work on an open source project and get paid for it.
+If you're interested participating in the program in 2022 with Fortran-lang,
+don't hesitate to contact us and we'll guide you toward applying.
+
+## Conferences and papers
+
+### FortranCon 2021
+
+Like in 2020, the Fortran event of this year was
+[FortranCon 2021](https://tcevents.chem.uzh.ch/event/14/), the international
+Fortran conference, held on September 23-24.
+The keynote speaker this year was [Damian Rouson](https://github.com/rouson),
+the head of the
+[Computer Languages and Systems Software (CLaSS)](https://crd.lbl.gov/divisions/amcr/computer-science-amcr/class/)
+group at the Lawrence Berkeley National Lab. 
+Fortran-lang had a dedicated session (a Fortran-lang minisymposium) on the
+second day of the conference, with the talks about specific Fortran-lang
+projects as well as the Google Summer of Code student presentations.
+FortranCon was hosted at the University of Zurich (UZH), and organized by
+[Tiziano Müller](https://github.com/dev-zero) from UZH and
+[Alfio Lazzaro](https://github.com/alazzaro) from Hewlett-Packard Enterprise.
+You can watch all FortranCon 2021 talks [here](https://www.youtube.com/playlist?list=PLeKbr7eYHjt5UaV9zQtY24oEbne9_uFni).
+
+### PackagingCon 2021
+
+Another big event for Fortran-lang was
+[PackagingCon 2021](https://packaging-con.org/), a conference dedicated to
+package management software and its community of developers.
+[Sebastian Ehlert](https://github.com/awvwgk) presented fpm--you can watch the
+recording below:
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YG8zEM1lAVM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</center>
+
+### Papers
+
+This year we also wrote our first paper documenting the motivation and goals of
+Fortran-lang: "Toward Modern Fortran Tooling and a Thriving Developer
+Community" by Milan Curcic, Ondřej Čertík, Brad Richardson, Sebastian Ehlert,
+Laurence Kedward, Arjen Markus, Ivan Pribec, and Jérémie Vandenplas
+([https://arxiv.org/abs/2109.07382](https://arxiv.org/abs/2109.07382)).
+
+<center>
+<a href="https://arxiv.org/abs/2109.07382">
+<img width="600" src="{{ site.baseurl }}/assets/img/Fortran-lang-2021-in-review/fortran-lang-paper.png">
+</a>
+</center>
+
+If you use any of the Fortran-lang projects for your publications,
```

Comment:
General comment: should be mention this in all README of all Fortran-lang projects?